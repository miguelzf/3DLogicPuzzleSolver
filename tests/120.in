- - - X P
- Y - X X
- - - - -
- - - - -
- X X - X

- - - - X
- - - - -
- - R P X
- Y G - X
- - - - -

X - X X -
- - - - -
- - R - -
- - - G -
- - - - -
