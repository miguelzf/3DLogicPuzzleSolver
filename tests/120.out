
Solution:
g  g  g  X  P  
g  Y  g  X  X  
g  y  g  g  g  
g  y  y  y  g  
g  X  X  y  X  
=============
g  y  y  y  X  ||  X  g  X  X  p  
g  y  r  r  r  ||  r  g  g  g  p  
g  y  R  P  X  ||  r  -  R  g  p  
g  Y  G  p  X  ||  r  r  r  G  p  
g  g  g  p  p  ||  p  p  p  p  p  
